cmd/kubectl-direct_csi/install.go

```
@@ -92,22 +93,20 @@ func install(ctx context.Context, args []string) (err error) {
 	}
 	defaultAuditDir, err := utils.GetDefaultAuditDir()
 	if err != nil {
-		return fmt.Errorf("unable to get default audit directory; %w", err)
+		klog.Errorf("unable to get default audit directory : %v", err)
 	}
 	if err := os.MkdirAll(defaultAuditDir, 0700); err != nil {
-		return err
+		klog.Errorf("unable to create default audit directory : %v", err)
```

As you just log and ignore the error, what is this indented for?